app/config/locale/translations/en.json

```diff
@@ -13,8 +13,9 @@
     "emails.magicSession.hello": "Hello,",
     "emails.magicSession.optionButton": "Click the button below to securely sign in to your {{project}} account. It will expire in 1 hour.",
     "emails.magicSession.buttonText": "Sign in to {{project}}",
-    "emails.magicSession.optionUrl": "If the button above doesn't show, use the following link:",
+    "emails.magicSession.optionUrl": "If you are unable to sign it using the button above, please visit the following link:",
```

Comment:
```suggestion
    "emails.magicSession.optionUrl": "If you are unable to sign in using the button above, please visit the following link:",
```

Reply:
Nooo 😭

Reply:
I know 😭

Reply:
And I am using GPT 4 to translate all, poor usage stats 😭